### ISSUE TITLE
Change Trajectory and Frame API to be consistent with HOOMD conventions

### ISSUE THREAD

**Issue by klywang** [glotzerlab/garnett#76]
To make glotzformats more user friendly, we should maintain a similar API to HOOMD's.
Things to change:
- `traj.positions` and `frame.positions` -> `traj.position` and `frame.position`
- `traj.orientations` and `frame.orientations` -> `traj.oritentaion` and `frame.orientation`
- `traj.type_ids` -> `traj.typeid` and `frame.typeid`
- `frame.velocities` -> `frame.velocity`

Task list:
- Add dummy `frame.typeid` property? (will be set properly in #75 - we can resolve the branches when merging or address this issue after #75 closed)
- Rename objects in `Frame` and `Trajectory` class
- Add deprecatoin warnings, but let users use old naming conventions (Do we want to have users add an import or line of code to use deprecated APIs? Or just let them access if but have glotzformats spit out a warning when they use it? I'm more inclined to the second option)
- Write/update unit tests

Extension of #73

**Comment by lyrivera** [comment]
Following the discussion in #73, I would add to this list:
- remove the current `types` attribute from both trajectory and frame objects
- rename `traj.type`->`traj.types` and add `frame.types`

**Comment by bdice** [comment]
We're going to split up the changes to the type system into a separate pull request.
